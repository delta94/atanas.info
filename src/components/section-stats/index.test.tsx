import * as React from 'react';
import { shallow } from 'enzyme';

import { SectionStats } from '.';

describe('SectionStats component', () => {
	it('Should render the SectionStats component', () => {
		const wrapper = shallow(<SectionStats />);

		expect(wrapper).toMatchSnapshot();
	});
});
