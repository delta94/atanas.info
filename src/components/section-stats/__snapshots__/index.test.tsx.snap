// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionStats component Should render the SectionStats component 1`] = `
<Component
  hasButton={true}
  hasShell={true}
  id="stats"
>
  <h1>
    Stats
  </h1>
  <div
    className="o-grid"
  >
    <div
      className="o-grid__item o-grid__item--1of2"
    >
      <h2>
        Commits (based on data by
         
        <Component
          href="https://sourcerer.io/scriptex"
        >
          Sourcerer
        </Component>
        )
      </h2>
      <ul>
        <li
          key="0"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/JavaScript-1112%20commits-yellow.svg"
            />
          </Component>
        </li>
        <li
          key="1"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/CSS-658%20commits-blue.svg"
            />
          </Component>
        </li>
        <li
          key="2"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/PHP-430%20commits-purple.svg"
            />
          </Component>
        </li>
        <li
          key="3"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/HTML-305%20commits-pink.svg"
            />
          </Component>
        </li>
        <li
          key="4"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/TypeScript-160%20commits-lightblue.svg"
            />
          </Component>
        </li>
        <li
          key="5"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/Wolfram-5%20commits-orange.svg"
            />
          </Component>
        </li>
        <li
          key="6"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/C-5%20commits-grey.svg"
            />
          </Component>
        </li>
        <li
          key="7"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/Gradle-5%20commits-red.svg"
            />
          </Component>
        </li>
        <li
          key="8"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/Java-5%20commits-green.svg"
            />
          </Component>
        </li>
        <li
          key="9"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/Objectiveâ€“C-5%20commits-lightgrey.svg"
            />
          </Component>
        </li>
        <li
          key="10"
        >
          <Component
            href="https://sourcerer.io/scriptex"
          >
            <img
              src="https://img.shields.io/badge/Shell-1%20commits-darkred.svg"
            />
          </Component>
        </li>
      </ul>
    </div>
    <div
      className="o-grid__item o-grid__item--1of2"
    >
      <h2>
        <a
          href="https://profile.codersrank.io/user/scriptex"
          rel="noopener noreferrer nofollow"
          target="_blank"
        >
          Codersrank
        </a>
         
        Profile
      </h2>
      <codersrank-widget
        username="scriptex"
      />
    </div>
  </div>
</Component>
`;
